.github/workflows/codeql.yml

```diff
@@ -7,6 +7,9 @@ on:
   schedule:
     - cron: '15 14 * * 1'
 
+permissions:
+  contents: read
```

Comment:
There is one job and it already has permissions defined. This has no value.

Reply:
But if the scorecard tool is so stupid that it lowers the score without it... so be it.

Reply:
Yes, unfortunately their automated check is not a very clever one but even if there is one job, defining a permission level is better to remove the possibility of misleading/mistaken results.